storage_service/static/js/project.js

```diff
@@ -27,4 +27,34 @@ $(document).ready(function() {
           },
       }
     });
-} );
+
+    $("a.request-delete").click(function() {
+        var self = $(this);
+        var uuid = self.data('package-uuid');
+        var pipeline = self.data('package-pipeline');
+        var $userDataEl = $("#user-data-packages");
+        var userId = $userDataEl.data('user-id');
+        var userEmail = $userDataEl.data('user-email');
+        var userUsername = $userDataEl.data('user-username');
+        var userAPIKey = $userDataEl.data('user-api-key');
+        var formData = {
+            "event_reason": "Storage Service user wants to delete AIP " + uuid + ".",
+            "pipeline": pipeline,
+            "user_id": userId,
+            "user_email": userEmail};
+        $.ajax({
+            type: "POST",
+            url: "/api/v2/file/" + uuid + '/delete_aip/',
```

Comment:
If you don't add the base url this wouldn't work when SS is deployed from a subpath right?

Reply:
Sure, I'll fix this to provide a full URL.